lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -51,6 +51,7 @@ export class RefreshTokenClient extends BaseClient {
             reqTimestamp,
             request.resourceRequestMethod,
             request.resourceRequestUri,
+            request.clientClaims,
```

Comment:
Also condense these into just passing `request` and destructure inside the function